src/core.js

```diff
@@ -49,7 +49,13 @@ function createModule( name, testEnvironment ) {
 		testsRun: 0,
 		unskippedTestsRun: 0,
 		childModules: [],
-		suiteReport: new SuiteReport( name, parentSuite )
+		suiteReport: new SuiteReport( name, parentSuite ),
+
+		// Pass along `skip` property from parent module, in case there is one,
+		// to childs. And use own otherwise.
+		// This property will be used to mark own tests and tests of child suites
+		// as `skipped`.
+		skip: parentModule !== null && parentModule.skip || modifiers.skip
```

Comment:
Just crossed my mind right now. What should be the order here ?
Should the modifier of the current module take precedence over the one from the parent module ?

One thing I forgot to test is what should happen if we defined an `only` module inside a `skipped/todo` one (I know this is rarely going to happen), or a `skipped` test inside a `todo` and vise versa ?

Right now, what I intend to implement is as follow:
* The tests of a normal module that was defined inside a `skipped/todo` will be treated as either `todo` to `skipped` based on the modifier of the parent module.
* If a module was defined as `only`, it will be the only module that will run. But, in case it is inside a `todo` or `skipped` module, its tests will be marked as either `todo` or `skipped` based on the modifier of the parent module.
* If a module was defined as `skipped` and is inside a `todo` module, its tests will be left intact.
* If a module was defined as `todo` and was inside a `skipped` module, its tests will be treated as `skipped` not as `todo`s.